static/src/js/components/Facets/Facets.js

```diff
@@ -66,35 +54,41 @@ const Facets = (props) => {
 
   const keywordsFacet = {
     title: 'Keywords',
-    changeHandler: changeCmrFacet,
+    changeHandler: cmrFacetHandler,
     options: {
       liftSelectedFacets: true
-    }
+    },
+    children: []
   }
 
   const platformsFacet = {
-    changeHandler: changeCmrFacet,
-    title: 'Platforms'
+    changeHandler: cmrFacetHandler,
```

Comment:
since `changeHandler` and `children` are the same in all of these, you could do something like 
```
const cmrFacetParent = {
	changeHandler: cmrFacetHandler,
	children: []
}
```
then...
```
const platformsFacet = {
	...cmrFacetParent,
	title: 'Platforms'
}
```
just a little bit less repeating yourself